qutebrowser/completion/completionwidget.py

```diff
@@ -89,6 +89,7 @@ class CompletionView(QTreeView):
     # https://github.com/The-Compiler/qutebrowser/issues/117
 
     resize_completion = pyqtSignal()
+    connected = None
```

Comment:
This shouldn't be here but `self.connected = None` in `__init__` instead, otherwise that'd be an attribute which is set this way in *every* instance of that class (i.e. a class rather than an instance variable).

I'd also say let's make it "private" (i.e. `_connected`) and please add a quick note about what it is to the docstring of the class.